src/status_im/subs/wallet/wallet.cljs

```diff
@@ -189,7 +188,7 @@
  :<- [:profile/currency]
  :<- [:profile/currency-symbol]
  (fn [[{:keys [tokens color]} currency currency-symbol]]
-   (mapv #(calc-token-value % color currency currency-symbol) tokens)))
+   (keep #(calc-token-value % color currency currency-symbol) tokens)))
```

Comment:
When using this sub (` :wallet/account-token-values`) the code in the view did:

`(filter identity sub-value)`

to remove `nil` values. Instead of doing that, we can use `keep`.

```
(filter identity (mapv some-fn tokens))
=
(keep some-fn tokens)
```

Reply:
@ulisesmac - I recently made changes to the `calc-token-value` function by removing the `when`.

My suggestion and plan is to filter the tokens (which have no symbol or no name) when saving them to app-db. This would help us resolve the token issues and make the subs readily usable on any views we want.

Reply:
Yes, that sounds as a good plan.
I'll rever this change then